integration_tests/tests/apply/prison_applications.cy.ts

```diff
@@ -4,24 +4,38 @@
 //
 //  Scenario: viewing the prison applications page as a referrer
 //      Given I am logged in as a prison bail referrer
+//      And there are submitted applications for my prison in the database
 //      When I visit the prison applications page
 //      Then I can see the prison applications page
+//      And I can see a list of applications for my prison
 
+import { applicationSummaryFactory } from '../../../server/testutils/factories'
 import PrisonApplicationsPage from '../../pages/apply/prisonApplicationsPage'
 import Page from '../../pages/page'
 
 context('Prison applications', () => {
   //  Scenario: viewing the home page as a referrer
   it('displays the prison applications page', () => {
     // Given I am logged in as a prison bail referrer
+    const prisonCode = 'ABC'
     cy.task('stubSignIn', { roles: ['ROLE_CAS2_PRISON_BAIL_REFERRER'] })
-    cy.task('stubAuthUser')
+    cy.task('stubAuthUser', { activeCaseLoadId: prisonCode })
```

Comment:
is this definitely the right casing? `caseLoad` feels strange to me vs `caseload`

Reply:
Yeah, it's the correct reference: https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-bail-ui/blob/main/server/data/hmppsAuthClient.ts#L7

You can do a search for other occurrences in the codebase.

Reply:
weird one but ok!